web-ng/src/app/components/project-header/project-header.component.html

```diff
@@ -22,6 +22,10 @@
       (focusOut)="updateProjectTitle($event)"
       class="inline-editor"
     ></inline-editor>
+    <button mat-stroked-button (click)="redirectMainPage()">
+      <mat-icon>list_alt</mat-icon>
```

Comment:
`redirect` is a misnomer here; `navigateTo` would be more appropriate. That said, click handlers are generally of the form `on*Click()`, in which case this one would be called `onProjectsButtonClick()`.

Reply:
updated